.github/workflows/build-and-test-conda.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and Test with conda
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os:
+        - ubuntu-latest
+        - macos-latest
+        # - windows-latest # FIXME not working
+        python:
+        - '3.10'
```

Comment:
Why have a single python version in a matrix? Are we planning to add more versions?

Reply:
It is in matrix instead of constant to add other versions if we start caring about it in the future